### ISSUE TITLE
standardize column names: case, abbreviation, etc

### ISSUE THREAD

**Issue by atn38** [lter/LTER-core-metabase#27]
eg: `id` in EMLUnitDictionary should be `UnitID` similar to `NameID`

eg: some inconsistencies in case, most PascalCase but some camelCase and some lowercase

**Comment by gastil** [comment]
Rules we follow today in re-naming columns of tables: In the lter_metabase schema, columns will have Pascal caps LikeThis in singular, not plural. Most columns already follow this convention. (There are a few EML controlled vocab tables which do not follow this pattern.)

Example: in DataSetAttributeEnumeration the columns code and definition are now Code and Definition.

The acronym ID such as in DataSetID, is both-caps, an exception to the Pascal caps. Even though in EML it is like packageId or userId, here in metabase we are going to W3D it as ID not Id.

See today's (2019-06-27) commit for the re-naming we did.

We opted not to tackle two tables which have bigger issues, covered elsewhere (in other git issues).
